include/openmc/settings.h

```diff
@@ -63,10 +63,12 @@ extern std::string path_source;
 extern std::string path_sourcepoint;      //!< path to a source file
 extern "C" std::string path_statepoint;   //!< path to a statepoint file
 
-extern "C" int32_t n_batches;      //!< number of (inactive+active) batches
-extern "C" int32_t n_inactive;     //!< number of inactive batches
-extern "C" int32_t gen_per_batch;  //!< number of generations per batch
-extern "C" int64_t n_particles;    //!< number of particles per generation
+extern "C" int32_t n_batches;                //!< number of (inactive+active) batches
+extern "C" int32_t n_inactive;               //!< number of inactive batches
+extern "C" int32_t n_max_lost_particles;     //!< maximum number of lost particles
```

Comment:
I would suggest removing the `n` to be consistent with the Python side:
```suggestion
extern "C" int32_t max_lost_particles;       //!< maximum number of lost particles
```

Reply:
No problem, will do